### DISCUSSION TITLE
J4-default-template Flavicon, Head and Body code insertion

### DISCUSSION THREAD

**Discussion by sluc69** [joomla/joomla-cms#34675] [2. Cold Features]
Since the Cassiopeia template has already been extended significantly (compared to J3 default template), Is it possible to add some fields in the 'style page' for insertion of code (links mostly) in the Head and Body. 

If I add in the index file it will be overridden on the next update of J4. (yes i know DON'T CHANGE THE FIXED ASSETS)
Or is there another permanent fix without adding an extension???

Adding a favicon choice would be nice. (without manually changing THE FIXED ASSETS!!)  ;-)

**Comment by dgrammatiko** [comment]
> Adding a favicon choice would be nice. 

Just copy your svg files into the `cassiopeia/images` folder and rename them to match the existing name. That's all

**Comment by dgrammatiko** [comment]
> If I add in the index file it will be overridden on the next update of J4. 

There are child templates that will allow you to update templates safely without losing your changes but the current templates are not implementing that (reason: the extension com_templates is not updated to support the new mode yet)
